project3.0/Blog.html

```diff
@@ -29,7 +29,7 @@
         </div>
     </header>
     <section class="section-blog" id="content">
-        <div class="blog-container">
+        <div class="container-blog">
```

Comment:
wrong.
You have this style:
```
.container {
    max-width: 856px;
    margin-left: auto;
    margin-right: auto;
}
```

So you can add new style: 
```
.container-blog {
max-width: 683px;
}
```

and add 2 class for div "container container-blog"